tests/fatfs-buffer.js

```diff
@@ -0,0 +1,79 @@
+//From https://github.com/vatesfr/xen-orchestra/blob/9f0b22d3e9c4672d96cb3f63b1e0fd42609e2406/packages/xo-server/src/fatfs-buffer.js
```

Comment:
Is this code restricted by the AGPL3 license then, or is its author willing to contribute it under the MIT/Apache terms of this library?

Reply:
I think I will replace it with example driver from readme and mkfs (https://superuser.com/a/668534)

Reply:
Fixed